### PR TITLE
P76.s replaced whitespace with newline at end of file

### PR DESCRIPTION
**Pull request by proudindiv**
I needed this for my process to generate a copy of the P76.agc file and for clean diffs.  I found a whole bunch of typo corrections like the ones you've been making while I was catching up to yours.  I'll submit them after this one is successful and I'm more confidant about using github.

### CODE REVIEW

**Comment by oldmud0** [comment]
This presents an interesting question: what kind of whitespace should be present at the end of a source file's content?
Also, these typos need to be checked against the original scans to determine whether or not they were transcription errors or actual (we call them "intentional") typos in source.

**Comment by proudindiv** [comment]
On this one, I checked the scanned page.  This PR should be applied for
consistency with the rest of the files.  As far as "white space" goes,
there were no control codes punched into the original Hollerith cards.  The
"tab" button on the card punch advanced the card in the punch to the next
set tab stop.  No "tab character" existed.  The passed over columns were
left blank, i.e. as a "space".  Almost every line in the files contains the
"tab character", and I'm not recommending that we change them at this
time.  I eventually may in my clone.  The cards were 80 columns wide, so
there should be trailing spaces to fill the cards out.  How we encode the
cards for the official repository is open for discussion.

Thanks for the response.

Gary Young
aka proudindiv

File: Comanche055/MEASUREMENT_INCORPORATION.agc [hunk 1]
```diff
@@ -362,7 +362,7 @@ FAZC		CALL
 			GRP2PC
 # Page 1259
 		VLOAD	VAD		# START 3RD PHASE OF INCORP2
-			X789		# 7TH,8TH,9TH,COMPONENT OF STATE VECTOR
```
Comment (oldmud0) [inline Comanche055/MEASUREMENT_INCORPORATION.agc]: See wopian's comment at #91:
> There are no spaces between 7th and COMPONENT - i.e 7TH,8TH,9TH,COMPONENT
> [(source)](http://www.ibiblio.org/apollo/ScansForConversion/Comanche055/1259.jpg)

File: Comanche055/JET_SELECTION_LOGIC.agc [hunk 2]
```diff
@@ -278,7 +278,7 @@ BDZCHECK	CA	ZNDX
 		BZF	TAUCHECK
 		
 ACRBDZ		CA	T5TEMP		# Z-TRANSLATION ACCEPTED EVEN THO WE MAY
-		TS	RWORD1		# HAVE INTRODUCED AN UNDESIREABLE ROLL
```
Comment (oldmud0) [inline Comanche055/JET_SELECTION_LOGIC.agc]: wopian at #91: Typo present in [source](http://www.ibiblio.org/apollo/ScansForConversion/Comanche055/1047.jpg).

File: Comanche055/INTERPRETER.agc [hunk 3]
```diff
@@ -1336,8 +1336,8 @@ UPPOS		XCH	L		# SAVE DECREMENTED UPPER PART.
 		AD	HALF
 		TS	A		# SKIPS ON OVERFLOW
 		TCF	+2
-		INCR	L		# RESTORE UPPER TO ROIGINAL VALUE
-		XCH	L		# SWAP A + L BACK.
```
Comment (oldmud0) [inline Comanche055/INTERPRETER.agc]: both rejected, see [source](http://www.ibiblio.org/apollo/ScansForConversion/Comanche055/1148.jpg)

File: Comanche055/INTERPRETER.agc [hunk 4]
```diff
@@ -2458,7 +2458,7 @@ DSQSUB		CA	MPAC +1		# SQUARES THE SCALAR CONTENTS OF MPAC.
 		XCH	MPAC +1
 		EXTEND
 		MP	MPAC
-		DDOUBL			# AND MAYBE OVEFLOW.
```
Comment (oldmud0) [inline Comanche055/INTERPRETER.agc]: approved, see [source](http://www.ibiblio.org/apollo/ScansForConversion/Comanche055/1183.jpg)

File: Comanche055/INTEGRATION_INITIALIZATION.agc [hunk 5]
```diff
@@ -97,7 +97,7 @@
 #
 # 2.2 SCALING
 #
-# THE INTEGRATION ROUTIEN WILL MAINTAIN THE PERMANENT MEMORY STATE VECTORS IN THE SCALING AND UNITS DEFINED IN
```
Comment (oldmud0) [inline Comanche055/INTEGRATION_INITIALIZATION.agc]: approved ([source](http://www.ibiblio.org/apollo/ScansForConversion/Comanche055/1310.jpg))

File: Comanche055/ERASABLE_ASSIGNMENTS.agc [hunk 6]
```diff
@@ -1561,7 +1561,7 @@ ESCAPE		EQUALS	DOTINC		# USED IN ARCSIN/ARCCOS.
 ENTRET		EQUALS	DOTINC		# EXIT FROM ENTER
 
 DOTRET		ERASE			# RETURN FROM DOT SUBROUTINE
-DVNORMCT	EQUALS	DOTRET		# DIVIDENT NORMALIZATION COUNT IN DDV.
```
Comment (oldmud0) [inline Comanche055/ERASABLE_ASSIGNMENTS.agc]: approved ([source](http://www.ibiblio.org/apollo/ScansForConversion/Comanche055/0070.jpg))

File: Comanche055/DOWN-TELEMETRY_PROGRAM.agc [hunk 7]
```diff
@@ -329,7 +329,7 @@ DNTMEXIT	EXTEND			# DOWN-TELEMETRY EXIT
 		CA	L		# RESPECTIVELY
 TMEXITL		EXTEND
 		WRITE	DNTM2
-TMRESUME	TCF	RESUME		# EXIT TELEMTRY PROGRAM VIA RESUME.
```
Comment (oldmud0) [inline Comanche055/DOWN-TELEMETRY_PROGRAM.agc]: approved ([source](http://www.ibiblio.org/apollo/ScansForConversion/Comanche055/1099.jpg))

File: Comanche055/P20-P25.agc [hunk 8]
```diff
@@ -531,7 +531,7 @@ V51NB		VN	5100
 #	7 REGISTER MRKBUF2 CONTAINING TIME2,TIME1,CDUY,OPTICS X,CDUZ, OPTICS Y,
 #	CDUX.
 #
-# ERASABLE INTIALIZATION REQUIRED
```
Comment (oldmud0) [inline Comanche055/P20-P25.agc]: approved ([source](http://www.ibiblio.org/apollo/ScansForConversion/Comanche055/1099.jpg))

File: Comanche055/P20-P25.agc [hunk 9]
```diff
@@ -782,7 +782,7 @@ PI/4.0		2DEC	.785398164
 #	MARKDATA, BASE ADDRESS OF MARK DATA
 #	REFSMMAT, ROTATION MATRIX FROM STABLE MEMBER TO BASIC REF. COORD. SYSTEM
 #
-# SUROUTINES CALLED:
```
Comment (oldmud0) [inline Comanche055/P20-P25.agc]: Approved, but colon should be replaced with hyphen (`SUBROUTINES CALLED-`) ([source](http://www.ibiblio.org/apollo/ScansForConversion/Comanche055/0578.jpg))

File: Comanche055/P32-P33_P72-P73.agc [hunk 10]
```diff
@@ -39,7 +39,7 @@
 # COELLIPTIC SEQUENCE INITIATION (CSI) PROGRAMS (P32 AND P72)
 #
 # MOD NO -1		LOG SECTION -- P32-P35, P72-P75
-# MOD BY WHITE. P	DATE 1JUNE67
```
Comment (oldmud0) [inline Comanche055/P32-P33_P72-P73.agc]: Rejected. Should be `WHITE.P` without spaces ([source](http://www.ibiblio.org/apollo/ScansForConversion/Comanche055/0649.jpg))

File: Comanche055/P34-35_P74-75.agc [hunk 11]
```diff
@@ -1419,7 +1419,7 @@ RTRNMU		STORE	RTMU
 # INPUT
 #	(1)	RVEC	POSITION VECTOR IN METERS
 #			SCALE FACTOR -- EARTH +29, MOON +27
-#	(2)	VVEC	VELOCITY VECTORY IN METERS/CENTISECOND
```
Comment (oldmud0) [inline Comanche055/P34-35_P74-75.agc]: Approved ([source](http://www.ibiblio.org/apollo/ScansForConversion/Comanche055/0495.jpg))

File: Comanche055/P51-P53.agc [hunk 12]
```diff
@@ -2105,7 +2105,7 @@ R56A1		CAF	VB53		# DISPLAY V53 REQUESTING ALTERNATE MARK
 # Page 782
 		CADR	GOMARK2
 		TCF	GOTOP00H	# V34:  TERMINATE
-		TCF	R56A1		# V33:  DON'T PROCEED -- JUST ENTER TO MARK
```
Comment (oldmud0) [inline Comanche055/P51-P53.agc]: Approved ([source](http://www.ibiblio.org/apollo/ScansForConversion/Comanche055/0782.jpg))

File: Comanche055/P51-P53.agc [hunk 13]
```diff
@@ -318,7 +318,7 @@ V06N89*		VN	0689
 # FUNCTION -- TO DISPLAY THE LANDING SITE LATITUDE,
 # LONGITUDE AND ALTITUDE.  TO ACCEPT NEW DATA VIA
 # THE KEYBOARD.  TO COMPUTE THE LANDING SITE
-# ORIENTATION FOR P52 OR P54
```
Comment (oldmud0) [inline Comanche055/P51-P53.agc]: Rejected ([source](http://www.ibiblio.org/apollo/ScansForConversion/Comanche055/0742.jpg))

File: Comanche055/PINBALL_GAME_BUTTONS_AND_LIGHTS.agc [hunk 14]
```diff
@@ -1358,7 +1358,7 @@ OPDEGOUT	CCS	MPAC		# RANGE IS 90 DEG
 		TC	NEGOPT		# IF NEG NON ZERO
 		AD	NEG1		# IF NEG ZERO SUBTRACT 1
 		AD	20BIAS
-BIASCOM		TS	MPAC		# TEST FOR OVEFLOW
```
Comment (oldmud0) [inline Comanche055/PINBALL_GAME_BUTTONS_AND_LIGHTS.agc]: Approved ([source](http://www.ibiblio.org/apollo/ScansForConversion/Comanche055/0335.jpg))

File: Comanche055/PINBALL_GAME_BUTTONS_AND_LIGHTS.agc [hunk 15]
```diff
@@ -2187,7 +2187,7 @@ HMSIN		TC	ALL3DEC		# IF ALL 3 WORDS WERE NOT LOADED, ALARM.
 		MP	MINCON		# LEAVES MINUTES CONTRIBUTION IN A,L
 # Page 353		
 		DAS	HITEMIN		# ADD IN MINUTES CONTRIBUTION
-		EXTEND			# IF THIS DAS OVEFLOWS, G/ 745 HR, 39MIN
```
Comment (oldmud0) [inline Comanche055/PINBALL_GAME_BUTTONS_AND_LIGHTS.agc]: This section of comments seem to be off by one line, but I will allow it. ([source](http://www.ibiblio.org/apollo/ScansForConversion/Comanche055/0353.jpg))

File: Comanche055/RT8_OP_CODES.agc [hunk 16]
```diff
@@ -171,7 +171,7 @@ CDUINC		TS	TEM2		# 1'S COMPL. QUANT. ARRIVES IN ACC.  STORE IT
 		AD	ONE
 		TCF	+4
 		AD	ONE
-		AD	ONE		# OVEFLOW HERE IF 2'S COMPL. IS 180 DEG.
```
Comment (oldmud0) [inline Comanche055/RT8_OP_CODES.agc]: [Approved](http://www.ibiblio.org/apollo/ScansForConversion/Comanche055/1511.jpg)

File: Comanche055/RT8_OP_CODES.agc [hunk 17]
```diff
@@ -250,7 +250,7 @@ TRANSP2		CAF	XNB1EB
 #	2.	GENERATE IN MPAC THE SIGNUM FUNCTION OF MPAC:
 #		ENTRY:		RTB
 #					SIGNMPAC
-# IN EITHER CASE, RETURN IS TO TEH NEXT INTERPRETIVE INSTRUCTION IN THE CALLING SEQUENCE.
```
Comment (oldmud0) [inline Comanche055/RT8_OP_CODES.agc]: There should be an extra space between TO and THE. ([source](http://www.ibiblio.org/apollo/ScansForConversion/Comanche055/1514.jpg))

File: Comanche055/TVCINITIALIZE.agc [hunk 18]
```diff
@@ -199,7 +199,7 @@ ATTINIT		CAE	DAPDATR1	# ATTITUDE-ERROR INITIALIZATION LOGIC
 	+8	CCS 	TTMP1		#	TEST TEMPORARY COUNTER
 		TCF	ATTINIT +5	#	BACK TO REPEAT FOR PITCH ERROR
 
-		CA	ERRBTMP		# ERROS ESTABLISHED AND LIMITED
```
Comment (oldmud0) [inline Comanche055/TVCINITIALIZE.agc]: [Approved](http://www.ibiblio.org/apollo/ScansForConversion/Comanche055/0940.jpg)

File: Comanche055/TVCROLLDAP.agc [hunk 19]
```diff
@@ -123,7 +123,7 @@
 #	CONTROL ACCELERATION CONACC, OR ITS RECIPROCAL, 1/CONACC, THE TVC
 #	ROLL DAP GAIN  (SEE TVCEXECUTIVE VARIABLE GAIN PACKAGE).  JET
 #	FIRING TIME IS SIMPLY THAT REQUIRED TO ACHIEVE THE DESIRED OGARATE,
-#	SUBJECT TO TEH LIMITATIONS DISCUSSED UNDER FUNCTIONAL DESCRIPTION,
```
Comment (oldmud0) [inline Comanche055/TVCROLLDAP.agc]: [Approved](http://www.ibiblio.org/apollo/ScansForConversion/Comanche055/0986.jpg)

File: Comanche055/TVCROLLDAP.agc [hunk 20]
```diff
@@ -150,7 +150,7 @@
 #	SECOND INTERVALS) WILL BENEFIT FROM THE CONVERGENT NATURE OF THE
 #	APPROXIMATION.
 #
-# FOR LARGE OGAERROR THE TANGENT INTERSECTS +-MINLIM SWITCH BOUNDARY BEFORE
```
Comment (oldmud0) [inline Comanche055/TVCROLLDAP.agc]: [Approved](http://www.ibiblio.org/apollo/ScansForConversion/Comanche055/0987.jpg)

File: Comanche055/UPDATE_PROGRAM.agc [hunk 21]
```diff
@@ -381,7 +381,7 @@ TIMEDIDL	EXTEND
 
 TIMEDIDR	INHINT
 		CAF	ZERO
-		ZL			# PICK UP INCRMENTER (AND ZERO
```
Comment (oldmud0) [inline Comanche055/UPDATE_PROGRAM.agc]: No space between INCREMENTER and left parenthesis. ([source](http://www.ibiblio.org/apollo/ScansForConversion/Comanche055/1504.jpg))

File: Luminary099/P32-P35_P72-P75.agc [hunk 22]
```diff
@@ -30,7 +30,7 @@
 # COELLIPTIC SEQUENCE INITIATION (CSI) PROGRAMS (P32 AND P72)
 #
 # MOD NO -1		LOG SECTION -- P32-P35, P72-P75
-# MOD BY WHITE.P	DATE 1JUNE67
```
Comment (oldmud0) [inline Luminary099/P32-P35_P72-P75.agc]: [Rejected](http://www.ibiblio.org/apollo/ScansForConversion/Luminary099/0618.jpg)

File: Luminary099/RTB_OP_CODES.agc [hunk 23]
```diff
@@ -129,7 +129,7 @@ CDUINC		TS	TEM2		# 1S COMPL.QUANT. ARRIVES IN ACC.  STORE IT
 		TCF	+4
 		AD	ONE
 # Page 1399
-		AD	ONE		# OVEFLOW HERE IF 2S COMPL. IS 180 DEG.
```
Comment (oldmud0) [inline Luminary099/RTB_OP_CODES.agc]: [Approved](http://www.ibiblio.org/apollo/ScansForConversion/Luminary099/1399.jpg)

File: Luminary099/RTB_OP_CODES.agc [hunk 24]
```diff
@@ -167,7 +167,7 @@ PULSEIMU	INDEX	FIXLOC		# ADDRESS OF GYRO COMMANDS SHOULD BE IN X1
 #	2.	GENERATE IN MPAC THE SIGNUM FUNCTION OF MPAC:
 #		ENTRY:		RTB
 #					SIGNMPAC
-# IN EITHER CASE, RETURN IS TO TEH NEXT INTERPRETIVE INSTRUCTION IN THE CALLING SEQUENCE.
```
Comment (oldmud0) [inline Luminary099/RTB_OP_CODES.agc]: Two spaces between TO and THE ([source](http://www.ibiblio.org/apollo/ScansForConversion/Luminary099/1401.jpg))

**Comment by oldmud0** [comment]
All changes reviewed.

**Pull request closed without merging**